### PR TITLE
Update trivyignore list for non-critical to us issues

### PR DESCRIPTION
**Pull request by ashb**
And update httpx for 2.1.4 -- a fixable security issue

### CODE REVIEW

**Review by kaxil — APPROVED** [review]

File: 1.10.15/buster/trivyignore [hunk 1]
```diff
@@ -22,3 +22,6 @@ CVE-2021-41265
 # In addition, this DOS vulnerability can really only cause
 # service disruption, so the risk is greater to do the upgrade
 CVE-2021-33503
+
+# Can't upgrade pyJWT.
+CVE-2022-292217
```
Comment (kaxil) [inline 1.10.15/buster/trivyignore:27]:
```suggestion
CVE-2022-29217
```

File: 2.2.5/bullseye/trivyignore [hunk 2]
```diff
@@ -2,3 +2,6 @@
 # and Kubernetes Python client is already using safe_load
 # Details: https://github.com/astronomer/issues-airflow/issues/39
 CVE-2020-1747
+
+# Can't upgrade pyJWT.
+CVE-2022-292217
```
Comment (kaxil) [inline 2.2.5/bullseye/trivyignore:7]:
```suggestion
CVE-2022-29217
```

File: 2.3.0/bullseye/trivyignore [hunk 3]
```diff
@@ -2,3 +2,6 @@
 # and Kubernetes Python client is already using safe_load
 # Details: https://github.com/astronomer/issues-airflow/issues/39
 CVE-2020-1747
+
+# Can't upgrade pyJWT.
+CVE-2022-292217
```
Comment (kaxil) [inline 2.3.0/bullseye/trivyignore:7]:
```suggestion
CVE-2022-29217
```

File: main/bullseye/trivyignore [hunk 4]
```diff
@@ -2,3 +2,6 @@
 # and Kubernetes Python client is already using safe_load
 # Details: https://github.com/astronomer/issues-airflow/issues/39
 CVE-2020-1747
+
+# Can't upgrade pyJWT.
+CVE-2022-292217
```
Comment (kaxil) [inline main/bullseye/trivyignore:7]:
```suggestion
CVE-2022-29217
```

**Review by kaxil — APPROVED** [review]

File: 2.1.4/buster/build-time-pip-constraints.txt [hunk 5]
```diff
@@ -268,7 +268,7 @@ hdfs==2.6.0
 hmsclient==0.1.1
 httpcore==0.13.7
 httplib2==0.19.1
-httpx==0.19.0
+httpx==0.20.0
```
Comment (ephraimbuddy) [inline 2.1.4/buster/build-time-pip-constraints.txt:271]: Which CVE made us update this? I would like to add it to the change log. No?

**Pull request merged**